lib/tasks/import_data_to_bigquery.rake

```diff
@@ -1,7 +1,32 @@
 DEFAULT_BATCH_SIZE = 200
 SLEEP_TIME = 2
+IRREGULAR_TABLE_NAMES = {
+  provider_relationship_permissions: 'ProviderRelationshipPermissions',
+  provider_user_notifications: 'ProviderUserNotificationPreferences',
+  provider_users_providers: 'ProviderPermissions',
+  references: 'ApplicationReference',
+}.freeze
+
+def entity_models
+  Rails.configuration.analytics.deep_merge(Rails.configuration.analytics_pii).map do |table_name, _|
```

Comment:
No need to deep-merge here — analytics contains all fields and pii contains a subset